src/index.html

```diff
@@ -11,7 +11,165 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post">
+      <fieldset class="form-section">
+        <legend>Personal information</legend>
+        <label class="form-field">
+        Surname:
+          <input
+          type="text"
+          name="Surname"
+          required
+          autocomplete="off"
+          >
```

Comment:
```suggestion
          <input
            type="text"
            name="Surname"
            required
            autocomplete="off"
          >
```

check it for whole page